config/schema/field_definitions.json

```diff
@@ -715,5 +715,45 @@
   "destination_country": {
     "description": "The country being exported to",
     "type": "identifiers"
+  },
+
+  "sector_business_area": {
+    "description": "The sector or business area",
+    "type": "identifiers"
+  },
+
+  "employ_eu_citizens": {
+    "description": "Whether or not the EU citizens are employed",
+    "type": "identifiers"
+  },
+
+  "doing_business_in_the_eu": {
+    "description": "Whether or not the going concern does business in the EU and they type of business (e.g. buying, selling, transporting)",
```

Comment:
Is this meant to say "their type of business"?

Reply:
👍 good spot - fixed in 24ef758